internal/connector/internal/config/connectors.go

```diff
@@ -62,6 +63,7 @@ func (c *ConnectorsConfig) AddFlags(fs *pflag.FlagSet) {
 	fs.StringArrayVar(&c.ConnectorEvalOrganizations, "connector-eval-organizations", c.ConnectorEvalOrganizations, "Connector eval organization IDs")
 	fs.BoolVar(&c.ConnectorNamespaceLifecycleAPI, "connector-namespace-lifecycle-api", c.ConnectorNamespaceLifecycleAPI, "Enable APIs to create, update, delete non-eval Namespaces")
 	fs.BoolVar(&c.ConnectorEnableUnassignedConnectors, "connector-enable-unassigned-connectors", c.ConnectorEnableUnassignedConnectors, "Enable support for 'unassigned' state for Connectors")
+	fs.StringArrayVar(&c.ConnectorsSupportedChannels, "connectors-supported-channels", []string{"stable"}, "Channels which connectors are visible")
```

Comment:
@lgarciaaco a default value of channel `stable` would mean that all devs and non prod envs would have to set this new variable. Instead let's make it so that the default empty value permits all channels to be supported. 

Also, please change the description text to `Connector channels that are visible`.